projects/igniteui-angular/src/lib/grids/filtering/excel-style/excel-style-conditional-filter.component.ts

```diff
@@ -40,6 +40,10 @@ export class IgxExcelStyleConditionalFilterComponent implements OnDestroy {
     @ViewChild('subMenu', { read: IgxDropDownComponent })
     public subMenu: IgxDropDownComponent;
 
+    public get filterNumber() {
```

Comment:
For some inexplicable reason `IgxExcelStyleConditionalFilterComponent` is exported in the public API https://www.infragistics.com/products/ignite-ui-angular/docs/typescript/latest/classes/IgxExcelStyleConditionalFilterComponent.html
So for such component, the general rule - either document API meant to be public _or_ in this case, unless I'm missing a need for this to be used externally, just mark it `protected` which has been usable in templates for a while now.